### ISSUE TITLE
Can sql based refreshKey check interval be changed

### ISSUE THREAD

**Issue by cndev123** [cube-js/cube.js#6303]
On this page https://cube.dev/docs/caching#in-memory-cache-refresh-keys it says either time based refresh or query based refresh is possible:

```
  // This refreshKey tells Cube to refresh data every 5 minutes
  refreshKey: {
    every: `5 minute`,
  },

  // With this refreshKey Cube will only refresh the data if
  // the value of previous MAX(created_at) changed
  // By default Cube will check this refreshKey every 10 seconds
  refreshKey: {
    sql: `SELECT MAX(created_at) FROM orders`,
  },
```

If I select query based refresh, **how can I adapt how often cube will check the refreshKey**, i.e. referring to this statement

>   // By default Cube will check this refreshKey every 10 seconds

**Comment by cndev123** [comment]
Can I just combine the two to get the effect?

```
refreshKey: {
    sql: `SELECT MAX(created_at) FROM orders`,
    every: `5 minute`,
  },
```

**Comment by igorlukanin** [comment]
Hi @cndev123 👋 

`queryCacheOptions.refreshKeyRenewalThreshold` is the [config option](https://cube.dev/docs/config#options-reference-orchestrator-options) you're looking for. However, please tune it carefully.

Is a 10-second interval too short or too long for your use case?

**Comment by paveltiunov** [comment]
@cndev123 Yep. You can combine those. That's the right way of setting the refresh key renewing intervals.

**Comment by cndev123** [comment]
@paveltiunov and @igorlukanin , thank you for your answers. Are both working or ...?
